internal/bundle/plain.go

```diff
@@ -14,8 +14,9 @@ type PlainV0 struct {
 	Objects []client.Object
 }
 
+const manifestsDir = "manifests"
```

Comment:
nit: this is a variable that applies to the whole bundle package, so if its going to be a global constant, i'd name it something like `plainv0ManifestsDir`